rocAL/source/augmentations/geometry_augmentations/node_crop_resize.cpp

```diff
@@ -81,15 +93,31 @@ void CropResizeNode::update_node() {
 }
 
 void CropResizeNode::init(float area, float aspect_ratio, float x_center_drift, float y_center_drift) {
+    _crop_param = std::make_shared<RocalRandomCropParam>(_batch_size);
     _crop_param->set_area_factor(ParameterFactory::instance()->create_single_value_param(area));
     _crop_param->set_aspect_ratio(ParameterFactory::instance()->create_single_value_param(aspect_ratio));
     _crop_param->set_x_drift_factor(ParameterFactory::instance()->create_single_value_param(x_center_drift));
     _crop_param->set_y_drift_factor(ParameterFactory::instance()->create_single_value_param(y_center_drift));
 }
 
 void CropResizeNode::init(FloatParam *area, FloatParam *aspect_ratio, FloatParam *x_center_drift, FloatParam *y_center_drift) {
```

Comment:
Why interpolation type is not added?

Reply:
As discussed offline, will not be changing the existing APIs and the init functions in this PR